templates/web/fixmystreet.com/header_extra.html

```diff
@@ -2,6 +2,10 @@
 <link rel="prefetch" href="[% version('/vendor/OpenLayers.Projection.OrdnanceSurvey.js') %]">
 [% END %]
 
+[% IF bodyclass.match('mappage') %]
```

Comment:
I'd have a `AND c.config.BASE_URL == "https://www.fixmystreet.com"` here too so it only happens on live, not development.